From c087c87bcaacd49a22d4a9e9588585ce730257a7 Mon Sep 17 00:00:00 2001
From: S-trace <S-trace@list.ru>
Date: Tue, 13 Jun 2017 18:10:36 +0300
Subject: [PATCH 9/9] [OTA] framework-res.apk: Add
 eu.chainfire.opendelta.intent.BROADCAST_STATE to protected broadcasts list

Fixes logcat warnings:
06-13 18:17:53.702  1300  2374 E ActivityManager: Sending non-protected broadcast eu.chainfire.opendelta.intent.BROADCAST_STATE from system 3476:eu.chainfire.opendelta/1000 pkg eu.chainfire.opendelta
06-13 18:17:53.702  1300  2374 E ActivityManager: java.lang.Throwable
06-13 18:17:53.702  1300  2374 E ActivityManager:       at com.android.server.am.ActivityManagerService.checkBroadcastFromSystem(ActivityManagerService.java:18120)
06-13 18:17:53.702  1300  2374 E ActivityManager:       at com.android.server.am.ActivityManagerService.broadcastIntentLocked(ActivityManagerService.java:18709)
06-13 18:17:53.702  1300  2374 E ActivityManager:       at com.android.server.am.ActivityManagerService.broadcastIntent(ActivityManagerService.java:18800)
06-13 18:17:53.702  1300  2374 E ActivityManager:       at android.app.ActivityManagerNative.onTransact(ActivityManagerNative.java:499)
06-13 18:17:53.702  1300  2374 E ActivityManager:       at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:2980)
06-13 18:17:53.702  1300  2374 E ActivityManager:       at android.os.Binder.execTransact(Binder.java:565)

Change-Id: I19f5ef066226f1033a0337107cf4d29d220e662a
---
 core/res/AndroidManifest.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/res/AndroidManifest.xml b/core/res/AndroidManifest.xml
index a52480f3a3f..eb4e7ed62f8 100644
--- a/core/res/AndroidManifest.xml
+++ b/core/res/AndroidManifest.xml
@@ -275,6 +275,7 @@
     <protected-broadcast android:name="android.intent.action.PROXY_CHANGE" />
 
     <protected-broadcast android:name="android.os.UpdateLock.UPDATE_LOCK_CHANGED" />
+    <protected-broadcast android:name="eu.chainfire.opendelta.intent.BROADCAST_STATE" />
 
     <protected-broadcast android:name="android.intent.action.DREAMING_STARTED" />
     <protected-broadcast android:name="android.intent.action.DREAMING_STOPPED" />
@@ -515,6 +516,7 @@
     <protected-broadcast android:name="android.telecom.action.TTY_PREFERRED_MODE_CHANGED" />
     <protected-broadcast android:name="android.telecom.action.CURRENT_TTY_MODE_CHANGED" />
     <protected-broadcast android:name="com.android.server.retaildemo.ACTION_RESET_DEMO" />
+    <protected-broadcast android:name="com.android.server.retaildemo.ACTION_RESET_DEMO" />
 
     <protected-broadcast android:name="cyanogenmod.intent.action.LID_STATE_CHANGED" />
     <!-- ====================================================================== -->
-- 
2.11.0

